app/assets/stylesheets/application.scss

```diff
@@ -13,3 +13,4 @@
  *= require_tree .
  *= require_self
  */
+ @import "bootstrap/scss/bootstrap";
```

Comment:
I think this may need to go at the top of the file to keep the sprockets gem functioning correctly.